packages/keycodes/src/test/index.js

```diff
@@ -230,3 +231,273 @@ describe( 'rawShortcut', () => {
 		} );
 	} );
 } );
+
+describe( 'isKeyboardEvent', () => {
+	afterEach( () => {
+		while ( document.body.firstChild ) {
+			document.body.removeChild( document.body.firstChild );
+		}
+	} );
+
+	function keyPress( target, modifiers = {} ) {
+		[ 'keydown', 'keypress', 'keyup' ].forEach( ( eventName ) => {
+			const event = new window.Event( eventName, { bubbles: true } );
+			Object.assign( event, modifiers );
+			target.dispatchEvent( event );
+		} );
+	}
+
+	function attachEventListeners( target, eventHandler ) {
+		[ 'keydown', 'keypress', 'keyup' ].forEach( ( eventName ) => {
+			target.addEventListener( eventName, eventHandler );
+		} );
+	}
+
+	describe( 'primary', () => {
+		it( 'should identify modifier key when Ctrl is pressed', ( done ) => {
+			const attachNode = document.createElement( 'div' );
+			document.body.appendChild( attachNode );
+			attachEventListeners( attachNode, ( event ) => {
+				expect( isKeyboardEvent.primary( event, undefined, isAppleOSFalse ) ).toBe( true );
+				done();
+			} );
+
+			keyPress( attachNode, {
+				ctrlKey: true,
+				key: 'Ctrl',
+			} );
+		} );
+
+		it( 'should identify modifier key when ⌘ is pressed', ( done ) => {
+			const attachNode = document.createElement( 'div' );
+			document.body.appendChild( attachNode );
+			attachEventListeners( attachNode, ( event ) => {
+				expect( isKeyboardEvent.primary( event, undefined, isAppleOSTrue ) ).toBe( true );
+				done();
+			} );
+
+			keyPress( attachNode, {
+				metaKey: true,
+				key: 'Meta',
+			} );
+		} );
+
+		it( 'should identify modifier key when Ctrl + M is pressed', ( done ) => {
+			const attachNode = document.createElement( 'div' );
+			document.body.appendChild( attachNode );
+			attachEventListeners( attachNode, ( event ) => {
```

Comment:
Is there any reason why:
```js
const attachNode = document.createElement( 'div' );
document.body.appendChild( attachNode );
```
couldn't be included inside `attachEventListeners`?

Reply:
The function was initially intended to mimic the generality of the keyPress function which was taken from another test. But i do see your point, I will update this function.